Sources/SQLKit/Builders/SQLCreateTriggerBuilder.swift

```diff
@@ -0,0 +1,73 @@
+extension SQLDatabase {
+    public func create(trigger: SQLExpression, table: SQLExpression, procedure: SQLExpression, when: SQLExpression, event: SQLExpression) -> SQLCreateTriggerBuilder {
+        .init(trigger: trigger, table: table, procedure: procedure, when: when, event: event, on: self)
+    }
+
+    public func create(trigger: String, table: String, procedure: String, when: SQLTriggerWhen, event: SQLTriggerEvent) -> SQLCreateTriggerBuilder {
```

Comment:
Nit: The more specific method should go above the less specific one. Reading a source file flows better if you get to more basal methods as you descend the file.